srcpkgs/clazy/template

```diff
@@ -4,8 +4,8 @@ version=1.11
 revision=3
 build_style=cmake
 build_helper=qemu
-hostmakedepends="llvm"
-makedepends="clang llvm17-devel clang-tools-extra"
+hostmakedepends="llvm17 perl"
+makedepends="llvm17-devel"
```

Comment:
Something about the shuffling around of components makes this template try to build a man-page when it didn't before.

Generating this man-page requires perl